### PR TITLE
modify fun fact list to approved fact list

### LINKED ISSUE #1727: Modify Fun Fact List to Approved Fact List
### Overview
We need to update the fun fact list to use the list of facts provided by PMs and approved by Bonnie

### Action Items
- [x] Replace list of strings in `data/fact.js` to match the facts provided in the document below

### Resources/Instructions
[Fun Facts List Document](https://docs.google.com/document/d/1ew6BaXb_H_WAbe1l2PFnP7wxLNzM92pVwnjegske32I/edit)

### PR DESCRIPTION
**Pull request by drakenguyen4000**
Fixes #1727
  - [x] Up to date with `main` branch
  - [ ] Branch name follows [guidelines](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md#feature-branching)
  - [ ] All PR Status checks are successful
  - [ ] Peer reviewed and approved

**Changes**
Updated fact list in data/fact.js to match the facts provided in word document.

### CODE REVIEW

File: data/facts.js [hunk 1]
```diff
@@ -2,16 +2,16 @@ import { shuffle } from '@utils';
...
+  'Hack for LA is a volunteer-driven civic tech organization and is a project of Civic Tech Structure, Inc.—an organization that empowers nonprofits and governments through innovative solutions, scaling, and experimentation, to drive positive change and community impact.', 
+  'Spanning 4,084 square miles, LA County is one of the largest in the United States.', 
+  'With nearly 10 million residents—accounting for 27% of California’s total population—LA County is the most populous of any county in the United States.', 
+  'LA County encompasses 88 cities and over 120 unincorporated communities.',
+  'Neighborhood Councils are funded by taxpayer dollars, and their board members, who are elected by local communities, serve as volunteer LA City government officials.',
+  'Each Neighborhood Council is distinct, with its own board structure and stakeholder representation. Board sizes range from 7 to 35 members, with terms typically lasting two years.',
+  'Neighborhood Councils receive $32,000 annually in public funds. These funds support community events, programs, and advocacy initiatives, addressing issues such as infrastructure, public safety, and economic development.',
```
Comment (Skydodle) [inline data/facts.js:14]: This should be with a '$' sign as '$29497.15'. The rest looks good to me. Pulled down and tested, all the new fun facts showed up as expected.

**Review by Skydodle — DISMISSED** [review]
Pulled down and tested, all new fun facts show up in the cycle as expected. Looks good to me!

**Review by Skydodle — APPROVED** [review]

**Review by ryanfchase — CHANGES_REQUESTED** [review]
@drakenguyen4000 Can you remove the extra spaces after some of these modified lines? And on lines 15-17 as well?

**Review by ryanfchase — APPROVED** [review]
Latest changes look good, approving

**Pull request merged**